03_launch_mgmt_cluster.sh

```diff
@@ -1,7 +1,9 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Why did this one need to change but not some of the others?

Reply:
It did not, that's a mess up, sorry. fixed and squashed